### DISCUSSION TITLE
Forever scrolling help text on new tab button - how to make it go away?

### DISCUSSION THREAD

**Discussion by edvin76** [piroor/treestyletab#3600] [General]
On my new tab button at the bottom there is a constant and super distracting text scrolling saying "dropping to outside of sidebar will create a link or bookmark at..."
Is there a way to make it go away? I couldn't find anything in the settings.

**Comment by irvinm** [comment]
I think what you want is this:

## In the options, "unlock expert options"
![image](https://github.com/user-attachments/assets/29d5d6b0-6eba-44e3-9914-fc61aaad1466)

## Then under "Drag and Drop", disable the notification
![image](https://github.com/user-attachments/assets/ade06aa2-997a-49ac-a52a-74ac6a23f469)

**Comment by edvin76** [comment]
Thank you, turning that setting on and off a couple times fixed it.

**Comment by pbhj** [comment]
I went to do this, closed TST by accident, then reopened it and the scrolling marquee text had gone. Yay.

**Comment by meedstrom** [comment]
Must be a bug, if it's only meant to show up while tab-dragging.  Happens to me too. I wonder how to track it down?

**Reply by piroor** [reply]
[Debug logs](https://github.com/piroor/treestyletab/wiki/How-to-inspect-tree-of-tabs#how-to-collect-debug-logs-of-tst) around the module "sidebar/drag-and-drop" will help our debugging. The notification message should be cleared with a log message `finishDrag from ...`. I think something blocks to run the operation and the log message is missing when you finished the drag-and-drop action, on your case.
